### ISSUE TITLE
 If there is already a media recorder running a recording, how do I cover a member who enters the room at the last moment?

### ISSUE THREAD

**Issue by LucasHimelfarb** [muaz-khan/MultiStreamsMixer#11]
Will there be any way to modify or push a new stream to the recorder? and what would happen inside a blob?

**Comment by luanraithz** [comment]
Doesn't the method `appendStreams` exist for that (appending stream to a running mixer)?

**Comment by chrismpalmer** [comment]
Yes this should be the method to use. I am using this but not getting events.

**Comment by numerized** [comment]
Hello, have you found the solution to this question @chrismpalmer ?
Thanks

**Comment by numerized** [comment]
Look at this solution, it worked for me!
What a relief
https://stackoverflow.com/questions/58166881/webrtc-conference-call-recording-using-recordrtc
